optaplanner-wb-webapp/src/main/resources/org/optaplanner/workbench/client/resources/i18n/AppConstants.properties

```diff
@@ -17,7 +16,6 @@ HomeProducer.ProjectAuthoring=Project Authoring
 HomeProducer.ArtifactRepository=Artifact Repository
 HomeProducer.Administration=Administration
 HomeProducer.DeployYourArtifacts=Deploy your Artifacts
-
```

Comment:
@ge0ffrey +1 for excluding the properties for now to keep logical groups visually separated.